storage-resize-images/functions/lib/config.js

```diff
@@ -20,5 +20,5 @@ exports.default = {
     cacheControlHeader: process.env.CACHE_CONTROL_HEADER,
     imageSizes: process.env.IMG_SIZES.split(","),
     resizedImagesPath: process.env.RESIZED_IMAGES_PATH,
-    deleteOriginalFile: process.env.DELETE_ORIGINAL_FILE,
+    deleteOriginalFile: process.env.DELETE_ORIGINAL_FILE === "true",
```

Comment:
Was this an unintentional change? It doesn't look like it is related to metadata.

Reply:
That will go away once all the branches are sync'ed. That was the artifact of a previous PR.

Reply:
That's due to the "next" branch having outdated "lib" files, which is what caused our rollback last Friday. Let's be double sure that the lib files in this PR are up to date before merging this time.